test/packaging/tar/xz_packaging.py

```diff
@@ -38,14 +41,75 @@
 if not tar:
     test.skip_test('tar not found, skipping test\n')
 
-# Windows 10 now supplies tar, but doesn't support xz compression
-# assume it's just okay to check for an xz command, because don't
-# want to probe the command itself to see what it supports
-xz = test.where_is('xz')
-if not xz:
-    test.skip_test('tar found, but helper xz not found, skipping test\n')
+if sys.platform == 'win32':
 
-xz_path = os.path.dirname(xz)
+    # Windows 10 and later supplies windows/system32/tar.exe (bsdtar).
+    # Not all versions support xz compression.  Check the version string
```

Comment:
Probable makes sense to move this logic into TestSCons?

Reply:
After copy/pasting/modifying from the xz source code it seemed like it would be a good idea to have only one parameterized implementation.

Wasn't sure where or what name to use.

Reply:
Updated to move common windows system tar code in bz2 and xz scripts to a private common module in the packaging/tar folder.

It should be easier now to move somewhere else if desired.  At present, only used in two places.

Reply:
Well that didn't work as I had hoped.  The source file `test/packaging/tar/_windows_tar.py` was "run" as a test (which passes) which is undesirable.

Seems unfortunate to have to add to TestSCons for two tests.